workshop/Lab2/README.md

```diff
@@ -71,30 +71,30 @@ A *replica* is how Kubernetes accomplishes scaling out a deployment. A replica i
 
 Kubernetes allows you to use a rollout to update an app deployment with a new Docker image.  This allows you to easily update the running image and also allows you to easily undo a rollout, if a problem is discovered after deployment.
 
-In the previous lab, we created an image with a `1` tag. Let's make a version of the image that includes new content and use a `2` tag. This lab also contains a `Dockerfile`. Let's build and push it up to our image registry.
+In the previous lab, we used an image with a `v1` tag. Let's make a version of the image that includes new content and use a `v2` tag. Let's pull the image from public registry and optionally push it up to our image registry.
 
 To update and roll back:
-1. Build the new docker image with a `2` tag:
+1. Pull the new docker image with a `v2` tag:
 
-   ```docker build --tag registry.ng.bluemix.net/<my_namespace>/hello-world:2 .```
+   ```docker pull ibmcom/guestbook:v2```
 
-2. Push the image to the IBM Cloud Container Registry:
+2. Push the image, optionally, to the IBM Cloud Container Registry:
```

Comment:
Do not optionally do anything. Do it or do not. What's in the script we run?

Reply:
The reason is that currently the only place where we showcase containerRegistry. There is no need for it and we can use the ibmcom image directly when deploy the Pod. thoughts?